tests/Dfe.PlanTech.Web.E2ETests/cypress/e2e/dynamic-page-validator/dpv-validators/answer-validator.js

```diff
@@ -0,0 +1,9 @@
+import { CleanText } from "../../../helpers/text-helpers";
+
+export const validateAnswers = (matchingQuestion) => {
```

Comment:
Another instance where destructuring the arguments might make it look cleaner, but generally a stylistic chocie.

Reply:
Done :)